src/MeshGroup.cpp

```diff
@@ -293,6 +293,11 @@ bool loadMeshIntoMeshGroup(MeshGroup* meshgroup, const char* filename, const Mat
             spdlog::info("loading '{}' took {:03.3f} seconds", filename, load_timer.restart());
             return true;
         }
+        else
+        {
+            spdlog::warn("loading '{}' failed", filename);
+            return false;
+        }
```

Comment:
redundant `else`

```suggestion
        spdlog::warn("loading '{}' failed", filename);
        return false;
```

Reply:
Is there a good reason for doing this ? I always wonder which one is better... Putting the `else` **is** indeed redundant, but it also makes sure you return something intentionally in all situations, even if you later add new `else if` conditions.

Reply:
https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

Reply:
Cool, thanks !